src/main/java/com/minecraftabnormals/abnormals_core/core/events/listener/ACEvents.java

```diff
@@ -1,35 +1,49 @@
-package com.minecraftabnormals.abnormals_core.core.events;
+package com.minecraftabnormals.abnormals_core.core.events.listener;
```

Comment:
Changing the package is a breaking change.

Reply:
Reverted/